test/cpp/interop/interop_client.cc

```diff
@@ -1052,6 +1052,32 @@ bool InteropClient::DoChannelSoakTest(int32_t soak_iterations) {
   return true;
 }
 
+bool InteropClient::DoLongConnectionTest(int32_t soak_iterations, 
+                                         int32_t iteration_interval) {
+  gpr_log(GPR_DEBUG, "Sending %d RPCs...", soak_iterations);
+  GPR_ASSERT(soak_iterations > 0);
+  GPR_ASSERT(iteration_interval > 0);
+  SimpleRequest request;
+  SimpleResponse response;
+  int num_failures = 0;
+  for (int i = 0; i < soak_iterations; ++i) {
```

Comment:
nit: just to keep this separated from the soak tests, can we use a separate flag to control the number of iterations taken that's special to this test?